docs/preview/index.md

```diff
@@ -21,6 +21,7 @@ For more granular packages we recommend reading the documentation.
 * Automate Azure Data Factory tasks ([powershell](featues/powershell/azure-data-factory))
 * Automate Azure Key Vault tasks ([powershell](features/powershell/azure-key-vault))
 * Automate Azure API Management tasks ([powershell](features/powershell/azure-api-management))
+* Automate Azure Table Storage tasks ([powershell](features/powershell/azure-table-storage))
```

Comment:
Should we create a `azure-table-storage` module, or a `azure-storage` module, which includes operations for table/blob/file/... (all sub-types of an Azure Storage Account)?

I'd tend to go for `azure-storage`, but perhaps that would become to general? 
What do you think: @tomkerkhove?

Reply:
Agree on `Arcus.Scripting.Storage.Table`
Also agree on adding on overlapping module `Arcus.Scripting.Storage`